packages/quill/src/app/screens/GenerateAccount/GenerateAccountAddress.tsx

```diff
@@ -0,0 +1,71 @@
+import { BlockieAddress, Body, Button, Heading, Tooltip } from '@mycrypto/ui';
+import { getGeneratedAccount, setGeneratedAccountPersistent, translateRaw } from '@quill/common';
+import React, { useState } from 'react';
+
+import type { IFlowComponentProps } from '@components';
+import { Box, Checkbox, Container, Link, Panel, PanelBottom } from '@components';
+import { DEFAULT_DERIVATION_PATH } from '@config/derivation';
+import { useDispatch, useSelector } from '@store';
+import { translate } from '@translations';
+
+export const GenerateAccountAddress = ({ flowHeader, onNext }: IFlowComponentProps) => {
+  const [showMnemonicPhrase, setShowMnemonicPhrase] = useState(false);
+  const { address, mnemonicPhrase, persistent } = useSelector(getGeneratedAccount);
+  const dispatch = useDispatch();
+
+  const handleToggle = () => dispatch(setGeneratedAccountPersistent(!persistent));
+  const handleClick = () => setShowMnemonicPhrase(!showMnemonicPhrase);
+
+  return (
+    <>
+      <Container>
+        {flowHeader}
+        <Box sx={{ textAlign: 'center' }} mb="3">
+          <Heading fontSize="24px" lineHeight="150%">
```

Comment:
Is there a theme value for this? And other font sizes in this file

Reply:
No, it seems to be between two values. We do it this way everywhere else in the app.